src/actions/addMany.js

```diff
@@ -33,13 +31,9 @@ export default co.wrap(function* (data, cfg, plop) {
 	for (let templateFile of templateFiles) {
 		const absTemplatePath = path.resolve(plop.getPlopfilePath(), templateFile);
 		const fileCfg = Object.assign({}, cfg, {
-			path: resolvePath(cfg.destination, templateFile, cfg.base)
+			path: resolvePath(cfg.destination, templateFile, cfg.base),
+			templateFile: absTemplatePath
 		});
-		Object.assign(fileCfg, (
```

Comment:
what is the purpose behind removing the `isBinary` check?

Reply:
It's simply not necessary if we pass the path of the template down to `addFile`, which itself check if the file is binary :) The only point of checking `isBinary` here was to avoid rendering the filename later in the `addFile`, and I moved this filename rendering responsibility to the addMany and add actions